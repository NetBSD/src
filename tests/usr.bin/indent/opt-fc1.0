/* $NetBSD: opt-fc1.0,v 1.3 2021/09/24 17:37:55 rillig Exp $ */
/* $FreeBSD$ */

/*
 * A comment
 * in column 1.
 *
 *
 *
 */

/* $ Neither indentation nor surrounding spaces. */
/*narrow*/

/* $ Indented by a single space, single spaces around the text. */
 /* space */

/* $ Indented by a single tab, single tabs around the text. */
	/*	tab	*/

/* $ The space between these comments gets removed. */
/* block1 */ /* block2 */

/* $ Both comment texts get surrounded by spaces. */
/*block1*//*block2*/
