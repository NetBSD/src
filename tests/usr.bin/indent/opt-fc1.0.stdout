/* $NetBSD: opt-fc1.0.stdout,v 1.3 2021/09/24 17:37:55 rillig Exp $ */
/* $FreeBSD$ */

/*
 * A comment in column 1.
 *
 *
 *
 */

/* $ The comment text got surrounded by spaces. */
/* narrow */

/* $ The indentation got removed. */
/* space */

/* $ The indentation got removed, only the leading tab got replaced by a space. */
/* tab	*/

/* $ The space between these comments got removed. */
/* block1 *//* block2 */

/* $ Both comment texts got surrounded by spaces. */
/* block1 *//* block2 */
